File: app/src/main/java/com/boardgamegeek/io/model/GeekListComment.java
package com.boardgamegeek.io.model;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;
import org.simpleframework.xml.Text;

@Root(name = "comment")
public class GeekListComment {
	@Attribute public String date;
	@Attribute public String editdate;
	@Attribute public String postdate;
	@Attribute public String thumbs;
	@Attribute public String username;
	@Text public String content;
}

File: app/src/main/java/com/boardgamegeek/entities/GeekListComment.kt
package com.boardgamegeek.entities

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class GeekListComment(
    val postDate: Long,
    val editDate: Long,
    val numberOfThumbs: Int,
    val username: String,
    val content: String
) : Parcelable
